packages/rlc-common/src/buildClient.ts

```diff
@@ -361,6 +361,11 @@ export function getClientFactoryBody(
         }
       });`;
   }
+
+  let apiVersionPolicyStatement = "";
+  if (model.apiVersionInfo?.definedPosition === "path") {
```

Comment:
we should remove this policy regardless if there is default value for the path api version.